src/numJewelsInStones.js

```diff
@@ -29,6 +29,15 @@
  */
 function numJewelsInStones(jewels, stones) {
   // write code here
+  if (jewels.length === 0) {
```

Comment:
если jewels.length вернет 0, то в булевом значении это false
```suggestion
  if (jewels.length) {
```